### DISCUSSION TITLE
Support for Leaderboards, Challenges, Achievements

### DISCUSSION THREAD

**Discussion by caseypugh** [fastlane/fastlane#18059] [Ideas]
For mobile games, would be massively helpful to manage game related content from fastlane for both Game Center and Google Play. When your game has 10-20 achievements, with images, in multiple languages it is a PITA to manage across both platforms. 

Google Play has an [API](https://developers.google.com/games/services/publishing/api/achievementConfigurations), but it might be difficult to implement Game Center? I started to dig into it with Charles proxy but it seems like they are using legacy software to maintain it, so it wasn't immediately clear to me if it would even be possible or not.

**Comment by pkclsoft** [comment]
Returning to fastlane after 9 years I'm amazed and frustrated to find that the metadata download/upload mechanisms still don't appear to support Game Center and all that go with that.  I agree with @caseypugh here.  I am prepping the launch of a game with 22 achievements, and 2 leaderboards.  I was really hoping fastlane would by now provide the ability to upload the metadata that is used by App Store Connect to manage these things.
